recipes/opsworks.rb

```diff
@@ -7,12 +7,27 @@
   include_recipe "cassandra-opsworks::datastax"
 end
 
-%w(cassandra.yaml cassandra-env.sh).each do |f|
-  template File.join(node["cassandra"]["conf_dir"], f) do
-    source "#{f}.erb"
+version=node[:package][:version].match(/_(.*)_/m)[1].strip[0...3]
+
+case version
+when "1.1"
+ config = "cassandra11.yaml"
+when "1.2", "2.0", "2.1"
+ config = "cassandra.yaml"
+end
```

Comment:
We should put an else statement here to catch the case where the version doesn't match one of the above. It should be a rare case, but we should guard against it.